packages/fxa-settings/src/components/Page2faReplaceRecoveryCodes/index.test.tsx

```diff
@@ -56,3 +75,42 @@ it('displays an error when fails to fetch new recovery codes', async () => {
   });
   expect(context.alertBarInfo?.error).toBeCalledTimes(1);
 });
+
+it('forces users to validate recovery code', async () => {
+  await renderPage2faReplaceRecoveryCodes();
+
+  await act(async () => {
```

Comment:
@jodyheavener will have <strike>opinions</strike> suggestions about this 😉

Reply:
yeah basically you shouldn't need to use `act` 99% of the time - [here's a presentation](https://drive.google.com/file/d/1-M_Dkh1gfJZNTJJYy_CEkDR9UkIp-Dak/view?usp=sharing) explaining why, as well as some other nifty tips